File: DatabaseMigrationSystem.Infrastructure/DbContext/Entities/UserToken.cs
namespace DatabaseMigrationSystem.Infrastructure.DbContext.Entities;

public class UserToken
{
    /// <summary>
    /// 
    /// </summary>
    public int UserId { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public string Token { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public DateTime Expires { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public DateTime Created { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public string CreatedByIp { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public DateTime Revoked { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public string RevokedByIp { get; set; }
    
    /// <summary>
    /// 
    /// </summary>
    public string ReplacedByToken { get; set; }

    public User User { get; set; }
}

File: front/Database.Migration.System/src/app/api/models/user-token.ts
/* tslint:disable */
/* eslint-disable */
import { User } from './user';
export interface UserToken {
  created?: string;
  createdByIp?: null | string;
  expires?: string;
  replacedByToken?: null | string;
  revoked?: string;
  revokedByIp?: null | string;
  token?: null | string;
  user?: User;
  userId?: number;
}
